cmd/users/main.go

```diff
@@ -313,28 +319,70 @@ func newService(db *sqlx.DB, tracer opentracing.Tracer, auth mainflux.AuthServic
 			Help:      "Total duration of requests in microseconds.",
 		}, []string{"method"}),
 	)
-	if err := createAdmin(svc, userRepo, c); err != nil {
+	if err := createAdmin(svc, userRepo, c, auth); err != nil {
 		logger.Error("failed to create admin user: " + err.Error())
 		os.Exit(1)
 	}
+
+	if c.adminCreatesUser != "" {
```

Comment:
It will be better if `c.adminCreatesUser` is compare with some constant string rather compare with `""` empty string
Example :
```suggestion
	if c.adminCreatesUser != "true" {
```